ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -187,8 +183,31 @@ tests:
     - ref: nutanix-upgrade-pre-config
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
-- as: vsphere-ipi-f28
-  cron: 28 6 18 * *
+- as: vsphere-agent-sno-f14
```

Comment:
1. Is @jiajliu 's comment resolved?
2. I remember agent team will add upgrade on their own, right?

Reply:
There are no other `sno` jobs better than this one:
```
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "- as: .*sno.*" ci-operator/config/openshift/openshift-tests-private
/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml
- as: aws-ipi-disc-priv-sno-basecap-none-f28
- as: aws-ipi-disc-priv-sno-basecap-none-f28-destructive
- as: vsphere-agent-disc-sno-f28
- as: vsphere-agent-sno-f28
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "- as: .*sno.*" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml
- as: aws-ipi-disc-priv-sno-basecap-none-f28
- as: aws-ipi-disc-priv-sno-basecap-none-f28-destructive
[jianl@jianl-thinkpadt14gen4 release]$ 
```

Reply:
Removing this one since we can not run `sno` jobs on eus upgrade.